scripts/ci.publish.js

```diff
@@ -49,7 +49,7 @@ function publishToNpm() {
     log('SKIP NPM is set: Lerna-publishing without publishing to NPM');
   }
 
-  exec.execSync(`lerna publish --cd-version "${versionType}" --yes --skip-git ${(dryRun || skipNpm) ? '--skip-npm' : ''}`);
+  exec.execSync(`lerna publish --cd-version "${versionType}" --yes --dist-tag ${npmTag} ${npmTag === 'latest'? '': '--preid=alpha'} --skip-git ${(dryRun || skipNpm) ? '--skip-npm' : ''}`);
```

Comment:
Are we not support to use `versionType=prerelease`?

Reply:
prerelease == prepatch == 10.16.0 -> 10.16.1-alpha.0
preminor == 10.16.0 -> 10.17.0-alpha.0
premajor == 10.16.0 -> 11.0.0-alpha.0

npm tags are set to branch name, unless stated otherwise in build parameters

Reply:
OK thanks! also 'noticed you updated this in the master-build's configuration on Jenkins 👍🏻 
What about the `alpha` part, which is static - wouldn't it make a fair use case that we would like to release `10.16.0-alpha.0` from branch `bigFeature1` and at the same time, `10.16.0-next.0` from branch `detoxNext`?

Reply:
was actually thinking the same thing :) We can improve on what we have of course

Reply:
I'll set the npm-tag as the prerelease tag